fixtures/app_with_framework_and_tests_with_custom_filenames/TuistConfig.swift

```diff
@@ -0,0 +1,8 @@
+import ProjectDescription
+
+let config = TuistConfig(
+  generationOptions: [
+    .generateManifest,
+    .xcodeProjectName("AwesomePrefix-\(TemplateString.Token.projectName)-AwesomeSuffix")
```

Comment:
Is this possible to write this as: `.xcodeProjectName("AwesomePrefix-\(.projectName)-AwesomeSuffix")`? Having to use the namespaces adds a lot of verbosity.